### PR TITLE
Add jellytitan keymap for Junco

### PR DESCRIPTION
**Pull request by JellyTitan**
## Description

Adding my custom keymap for the Junco.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/junco/keymaps/jellytitan/config.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Ryan Neff (@JellyTitan)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* - Encoder settings - */
+#ifdef ENCODER_ENABLE
+#define ENCODER_RESOLUTION 4
```
Comment (dunk2k) [inline keyboards/junco/keymaps/jellytitan/config.h:8]:
```suggestion
#    define ENCODER_RESOLUTION 4
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/junco/keymaps/jellytitan/keymap.c [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+ */
+#ifdef ENCODER_MAP_ENABLE
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    /* Left Hand */                             /* Right Hand */
+    /* Switch between tabs. (Control + Tab). */ /* Switch between open apps on Mac. (Command + Tab + timer logic) */
+    [_QWERTY] = {ENCODER_CCW_CW(C(KC_TAB), S(C(KC_TAB))), ENCODER_CCW_CW(CMD_TAB_CCW, CMD_TAB_CW)},
+    /* Scrolls left & right. (Shift + Mouse Wheel Up). */ /* Scrolls up and down. (Page Down & Page Up - mouse wheel scroll incraments are too small) */
+    [_LOWER] = {ENCODER_CCW_CW(S(KC_MS_WH_UP), S(KC_MS_WH_DOWN)), ENCODER_CCW_CW(KC_PGDN, KC_PGUP)},
+    /* Selects adjacent words. (Command + Shift + Right Arrow). */ /* Jumps to end/start of line. Hold shift to select. (Gui + arrow). */
+    [_RAISE] = {ENCODER_CCW_CW(C(S(KC_LEFT)), C(S(KC_RGHT))), ENCODER_CCW_CW(G(KC_LEFT), G(KC_RGHT))}};
```
[the remark this answers is not in the document]
Comment (JellyTitan) [reply keyboards/junco/keymaps/jellytitan/keymap.c:206]: Thank you!

**Comment by JellyTitan** [comment]
Thanks for the review! I appreciate your patience.

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/junco/keymaps/jellytitan/config.h [hunk 3]
```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Ryan Neff (@JellyTitan)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* - Encoder settings - */
+#ifdef ENCODER_ENABLE
+#    define ENCODER_RESOLUTION 4
+#endif
```
Comment (waffle87) [inline keyboards/junco/keymaps/jellytitan/config.h:9]: Default behaviour.
```suggestion
```

File: keyboards/junco/keymaps/jellytitan/config.h [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    - RGB Stuff -
+    All effects can be found in the QMK docs:
+    https://docs.qmk.fm/#/feature_rgb_matrix?id=rgb-matrix-effects
+*/
+#ifdef RGB_MATRIX_ENABLE
+
+/* Turn off RGB light when host is asleep. (Doesn't seem to work for mac)? */
+#    define RGBLIGHT_SLEEP
+#    define RGBLIGHT_LAYERS
+
```
Comment (waffle87) [inline keyboards/junco/keymaps/jellytitan/config.h:44]:
```suggestion
#    define RGB_DISABLE_WHEN_USB_SUSPENDED
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
Unfortunately since this PRs creation, QMK has [changed its stance](https://docs.qmk.fm/#/ChangeLog/20230827?id=notable-changes) on accepting user keymaps to the repository.

> qmk/qmk_firmware is no longer accepting PRs for keymaps other than for manufacturer-supported keymaps. User keymap workflow has been documented [here](https://docs.qmk.fm/#/newbs) for several years. This change is to progressively reduce the maintenance burden on the project, and to allow us to focus on the core features of QMK.

In the near future we will be providing a more user focused workflow for keymaps, but till then the suggestion is to keep changes in your own fork.

While we appreciate the time that was taken on this PR, hopefully you understand we are closing it, so that QMK can improve in the long term.

**Pull request closed without merging**